### DISCUSSION TITLE
Scenes loading black & blue

### DISCUSSION THREAD

**Discussion by ro6ertkenny** [mozilla/hubs#3348] [Hubs Cloud Questions] [answered]
I have two Enterprise Hubs stacks running for past couple months - about a week ago one of the stacks started getting and intermittent scene loading issue where the initial scene had a dark blue plane and a black background ... and nothing else in the scene - it would allow you to load in but nothing would appear.  I tried changing scenes and every scene I would try to load would have the same problem.  I tried clearing cache on browser and it fixed the problem the first time but then it didn't work again after that.  I tried on multiple devices with multiple browsers (Desktop/Ubuntu 20.04 Firefox & Chrome, Android phone & tablet - Firefox & Chrome, Quest 1 & 2 - Oculus Browser)

The second stack I have is identical - both are c4.large w/one app server and one voice server - I compared the console on both and I don't see any discrepancies between the two - the second stack is still working and I haven't seen this issue occur yet (see attached screenshots)
![crossroads biind app](https://user-images.githubusercontent.com/22346309/99267977-988edb00-27f2-11eb-9636-6b609b8b001a.png)
![biind app](https://user-images.githubusercontent.com/22346309/99268040-ab091480-27f2-11eb-94b9-6e51c2841242.png)

Has anyone experienced this yet?

**Comment by johnshaughnessy** [comment]
The room in the top screenshot looks like it is in the loading environment.
![image](https://user-images.githubusercontent.com/4072106/99503798-7cdf1e00-2933-11eb-849b-654aee1a2a29.png)
This is the default environment when no scene is specified.

When you try to change scenes, are the scenes you are using from the same hubs cloud instance? (If so, you should be able to see them in the admin interface for the instance.) Can you attach the console log from when you try to change the scene? That might help us diagnose the issue.

**Reply by ro6ertkenny** [reply]
When I try to change scenes I am selecting from the featured scenes from the same hubs cloud instance.  I can see them in the admin console and when I launch them from there and open a new room ... the scene loads fine (??).   When I'm in the new room that I opened from launching the scene from the admin console ... I'm also able to change to a new scene with no problem.  

So the issue is when you try to create a new room from the landing page, (crossroads.biind.app) whatever default scene it picks will not load, and when I choose a new scene to load from there, the scene loads in completely black (without the blue plane).  I've attached the additional screenshots from the console when accessing the scenes from the admin console.
![crossroads biind app-initial_scene](https://user-images.githubusercontent.com/22346309/99534826-08799e80-2976-11eb-975b-4a8e81507b74.png)
![crossroads biind app-new_scene](https://user-images.githubusercontent.com/22346309/99534833-0adbf880-2976-11eb-8e2a-2c1baed55c1e.png)
![crossroads biind app-admin_console](https://user-images.githubusercontent.com/22346309/99534888-1e875f00-2976-11eb-9fb9-fbc5c3219a2f.png)
![crossroads biind app-admin_scene_change_console](https://user-images.githubusercontent.com/22346309/99535035-57bfcf00-2976-11eb-92ae-be8671f92846.png)

**Comment by ro6ertkenny** [comment, marked as answer]
**UPDATE** I went through the approved scenes and added a default tag to one of the scenes and now when you create a room from the landing page ... it loads the default scene in ... and I'm also able to change scenes now.  We'll make sure there is always a default scene tag on one of the approved scenes going forward ... however, it still doesn't make sense how we've been able to create a room with a randomly chosen scene from the landing page for the past couple months without a problem until about a week or so ago.  We didn't have any default tags in any of the approved scenes so I don't know why it was working if there is a requirement for a default scene tag, and if it's not required why did it stop working in the first place and then start working again when I added a default tag.
